modules/social_features/social_profile/modules/social_profile_privacy/social_profile_privacy.install

```diff
@@ -63,3 +65,178 @@ function _social_profile_privacy_get_permissions($role) {
 
   return $permissions[$role];
 }
+
+/**
+ * Define visibility options of separate fields per user.
+ */
+function social_profile_privacy_update_8001(&$sandbox) {
+  if (!isset($sandbox['total'])) {
+    /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $display */
+    $display = \Drupal::entityTypeManager()->getStorage('entity_form_display')
+      ->load('profile.profile.default');
+
+    foreach ($display->getThirdPartySettings('field_group') as $fields_group => $settings) {
+      $states = [];
+
+      foreach ($settings['children'] as $field) {
+        $states[$field] = SocialProfilePrivacyHelperInterface::HIDE;
+      }
+
+      $sandbox['fields'][$fields_group] = serialize($states);
```

Comment:
It's better to use "Serialization API" instead.
https://www.drupal.org/docs/8/api/serialization-api/serialization-api-overview

Reply:
Done.